modules/notification/indexer/indexer.go

```diff
@@ -148,3 +148,7 @@ func (r *indexerNotifier) NotifyIssueChangeContent(doer *models.User, issue *mod
 func (r *indexerNotifier) NotifyIssueChangeTitle(doer *models.User, issue *models.Issue, oldTitle string) {
 	issue_indexer.UpdateIssueIndexer(issue)
 }
+
+func (r *indexerNotifier) NotifyIssueChangeRef(doer *models.User, issue *models.Issue, oldRef string) {
+	issue_indexer.UpdateIssueIndexer(issue)
```

Comment:
Since the branch name will not be indexed, I don't think we need to do the reindex.

Reply:
Do I remove the reference to this function from all other files as well?
![image](https://user-images.githubusercontent.com/20846919/91126486-66168b80-e6a4-11ea-88a8-ab25bdc313f2.png)

Reply:
I think it's better to keep it in. Just because we don't currently doesn't mean we won't in future.